test/extensions/filters/http/ext_authz/ext_authz_integration_test.cc

```diff
@@ -40,14 +40,25 @@ class ExtAuthzGrpcIntegrationTest : public Grpc::GrpcClientIntegrationParamTest,
       setGrpcService(*proto_config_.mutable_grpc_service(), "ext_authz",
                      fake_upstreams_.back()->localAddress());
 
+      proto_config_.mutable_filter_enabled()->set_runtime_key("envoy.ext_authz.enable");
+      proto_config_.mutable_filter_enabled()->mutable_default_value()->set_numerator(100);
+      proto_config_.mutable_deny_at_disable()->set_runtime_key("envoy.ext_authz.deny_at_enable");
```

Comment:
Seems like it should be `envoy.ext_authz.deny_at_disable`?

Reply:
Could you add: `Runtime guard:` in the PR description above? Seems like we're adding one?

Reply:
Updated.